hax/hax/queue/__init__.py

```diff
@@ -86,11 +86,13 @@ def handle_ioq_stob_error(self, payload: Dict[str, Any]) -> None:
         ids: List[MessageId] = q.get()
         self.herald.wait_for_any(HaLinkMessagePromise(ids))
 
-    def to_ha_state(self, objinfo: dict) -> Optional[HAState]:
+    def to_ha_state(self, objinfo: Dict[str, str]) -> Optional[HAState]:
         try:
             sdev_fid = self.confobjutil.drive_to_sdev_fid(
                 objinfo['node'], objinfo['device'])
+            state = ServiceHealth.OK if objinfo[
```

Comment:
strange line break

Reply:
That's how [yapf](https://github.com/google/yapf) formatted the sources.